CI/physmon/phys_perf_mon.sh

```diff
@@ -66,5 +66,25 @@ run \
     -p $outdir/truth_tracking_plots \
 
 
+echo "::group::full_chain_odd"
+
+thirdparty/OpenDataDetector/ci/full_chain_odd.py -o $outdir
+
+build/bin/ActsAnalysisResidualsAndPulls \
+    --predicted --filtered --smoothed --silent \
+    -i $outdir/trackstates_ckf.root \
+    -o $outdir/acts_analysis_residuals_and_pulls.root \
+
+histcmp \
+    --title "full_chain_odd" \
+    --label-reference=$refcommit \
+    --label-monitored=$commit \
+    $outdir/acts_analysis_residuals_and_pulls.root \
+    CI/physmon/reference/acts_analysis_residuals_and_pulls.root \
```

Comment:
Is the HTML output too large?

Reply:
yeah it was like 200mb uncompressed and 100mb compressed

Reply:
Hmm. Maybe create it anyway and adjust the expiry time? Although it would be good to have this for a while in principle.

Reply:
I will leave it out for now. the root files for the plots are in the artifacts and could be plotted manually if necessary. but we can adopt that at any time if necessary